src/interpolations.jl

```diff
@@ -51,6 +51,11 @@ const InterpolationByDim{dim} = Interpolation{<:AbstractRefShape{dim}}
 abstract type ScalarInterpolation{refshape, order} <: Interpolation{refshape, order} end
 abstract type VectorInterpolation{vdim, refshape, order} <: Interpolation{refshape, order} end
 
+# Conformity, :H1, :Hdiv, :Hcurl, and :L2 currently implemented.
+struct Conformity{K}
+    Conformity(conformity::Symbol) = new{conformity}()
+end
```

Comment:
Out of curiosity: Is there a reason why we do this instead of four different structs?

Reply:
Seemed unnecessary to introduce the hierarchy below an `AbstractConformity` with several subtypes, when e.g. `Conformity{:H1}` reads quite good for the dispatch.

But could be changed too of course if that is considered better.

Reply:
I was not even thinking about a type hierarchy here and was just curious. For me having the types directly available looks a bit more stable for users since the type accepts typos, but for the short names here it should be okay and these should be easy to find. @fredrikekre do you see any obvious fallacies with this pattern?

Reply:
Doesn't really matter since there will be four different concrete structs either way.

Reply:
Decided to change, I agree that it looks more standard, and the possible typo is a reasonable concern. The common type was only used for tests.